src/client/components/Chart/Pie/Pie.tsx

```diff
@@ -15,6 +15,7 @@ const Pie = (props: Props) => {
   const { data } = props
 
   const { t } = useTranslation()
+  const totalValue = useMemo(() => data.reduce((sum, entry) => sum + entry.value, 0), [data])
```

Comment:
Recharts tooltip has no information about the percentage, so we calculate it using the totalValue